ztools/x509/pkix/json.go

```diff
@@ -200,6 +200,10 @@ func (o *OtherName) UnmarshalJSON(b []byte) error {
 		IsCompound: true,
 		Bytes:      oName.Value,
 	}
+	o.Value.FullBytes, err = asn1.Marshal(o.Value)
+	if err != nil{
```

Comment:
Also - this `if err != nil .... return nil` can be replaced by a simple `return`